admin/class-phpredis-purger.php

```diff
@@ -106,7 +106,7 @@ public function purge_url( $url, $feed = true ) {
 		/**
 		 * Filters the URL to be purged.
 		 *
-		 * @since 2.1.0
+		 * @since 2.0.4
```

Comment:
@pradeep910 Introducing of filters/actions is major change so I'd suggest changing plugin version to `2.1.0`.

Reply:
Ok changing to 2.1.0